sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/Transforms.java

```diff
@@ -224,6 +227,31 @@ static BlobHttpHeaders toBlobHttpHeaders(PathHttpHeaders pathHTTPHeaders) {
             .setContentMd5(pathHTTPHeaders.getContentMd5());
     }
 
+    static BlobInputStreamOptions toBlobInputStreamOptions(DataLakeFileInputStreamOptions options) {
```

Comment:
I'm surprised this class isn't in implementation package.

Reply:
I think it's because we could actually make it package protected here.

Reply:
I'll attribute this question to heatwave  then :-)